test/prototype_transforms_kernel_infos.py

```diff
@@ -2,13 +2,16 @@
 import functools
 import itertools
 import math
-from typing import Any, Callable, Dict, Iterable, Optional, Sequence
+from collections import defaultdict
+from typing import Any, Callable, Dict, Iterable, List, Optional, Sequence, Tuple
 
 import numpy as np
 import pytest
 import torch.testing
 import torchvision.ops
 import torchvision.prototype.transforms.functional as F
+
+from _pytest.mark.structures import MarkDecorator
```

Comment:
This is only needed for annotating the attributes of the data classes below. I'll send a follow-up PR removing them, since the upside of `@dataclasses.dataclass` is outweighed by the downside of importing from a private namespace.